lms/templates/instructor/edx_ace/addbetatester/email/body.html

```diff
@@ -5,62 +5,57 @@
 {% block content %}
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
     <tr>
-        <td>
-            <h1>
-                {% filter force_escape %}
-                {% blocktrans %}
-                    You have been invited to be a beta tester for {{ course_name }} at {{ site_name }}
-                {% endblocktrans %}
-                {% endfilter %}
-            </h1>
+        <td style="width: 100%;
+            max-width: 800px;
+            background-color: #FFF;
+            color: #000000;
+            margin: auto;
+            text-align: center;
+            border: 1px solid #F3F3F3;">
 
-            <p style="color: rgba(0,0,0,.75);">
-                {% filter force_escape %}
-                {% blocktrans %}The invitation has been sent by a member of the course staff.{% endblocktrans %}
-                {% endfilter %}
-                <br />
-            </p>
+            <img src="{% static 'pearson-pols-theme/images/PearsonLogo_Primary_Blk_RGB.png' %}" style="width: 150px; padding: 5px;" />
 
-            {% if auto_enroll %}
-                <p style="color: rgba(0,0,0,.75);">
-                    {% filter force_escape %}
-                    {% blocktrans %}To start accessing course materials, please visit:{% endblocktrans %}
-                    {% endfilter %}
+            <div style="width: 90%;
+                        background-color: #D4EAE4;
+                        text-align: left;
+                        padding: 2% 5%;
+                        background-image: url('{% static 'pearson-pols-theme/images/laptop_transparent-bg.png' %}');
+                        background-size: 50%;
+                        background-position: right bottom;
+                        background-repeat: no-repeat;">
+
+                <h1 style="font-family:'Playfair Display', serif; font-weight: 700;">
+                    {% blocktrans %}
+                    You have been invited to be a beta tester for {{ course_name }} at {{ site_name }}
+                {% endblocktrans %}
```

Comment:
@Squirrel18  bad indentation.

Reply:
There are many details in this template, this template is taken from the existing one, it's not the purpose of this PR to fix them, but to fix the main issue, however, applied.